src/viur/core/module.py

```diff
@@ -96,8 +96,10 @@ def parse_value_by_annotation(annotation: type, name: str, value: str | list | t
                 return float(value)
             elif annotation is bool:
                 return utils.parse.bool(value)
-            elif annotation is types.NoneType:
-                return None
+            elif annotation is types.NoneType or annotation is None:
+                if value in (None, "None", "null"):
```

Comment:
Should we turn this into something like with `utils.parse.bool`?

Reply:
It think this case is too special, I never needed this before.